internal/operator-controller/controllers/clusterextension_controller.go

```diff
@@ -97,6 +97,7 @@ type InstalledBundleGetter interface {
 //+kubebuilder:rbac:groups=core,resources=serviceaccounts/token,verbs=create
 //+kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get
 //+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles;clusterrolebindings;roles;rolebindings,verbs=list;watch
+//+kubebuilder:rbac:groups=*,resources=*,verbs=*
```

Comment:
Would not it result in OLM have all possible permissions, do we want that?

Reply:
that's an open question at the moment. But given that we can create tokens for any service account, that's essentially what we have already XDD